influxdb3_write/src/persister.rs

```diff
@@ -754,11 +757,11 @@ mod tests {
         }
         let snapshots = persister.load_snapshots(9500).await.unwrap();
         // We asked for the most recent 9500 so there should be 9001 of them
```

Comment:
This comment is stale now.

Reply:
Right, I bumped this down because I was running out of file descriptors on my machine.

Reply:
Ah yes, that kept getting me too, good call.